Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -293,6 +295,55 @@ final class PledgeViewModelTests: TestCase {
     }
   }
 
+  func testUpdateContext_ConfirmationLabelShowsTotalAmount() {
+    let dateComponents = DateComponents()
+      |> \.month .~ 11
+      |> \.day .~ 1
+      |> \.year .~ 2_019
+      |> \.timeZone .~ TimeZone.init(secondsFromGMT: 0)
```

Comment:
I know this code was already here, but could you get rid of the `.init`s here too?